### PR TITLE
feat: disconnect observers and add new URLs to report for web vitals

### PR DESCRIPTION
**Pull request by Fine0830**
Screenshot

Signed-off-by: Qiuxia Fan <qiuxiafan@apache.org>

### CODE REVIEW

**Comment by wu-sheng** [comment]
Could you share more context about what does this mean?

**Comment by Fine0830** [comment]
> Could you share more context about what does this mean?

Sure. After the agent obtains relevant data, disconnect all observers
and stop the `MutationObserver` instance from receiving further notifications until `observe()` is called again.

**Review by wu-sheng — DISMISSED** [review]

**Comment by Fine0830** [comment]
Since I batched reporting performance metrics, the percentile metrics have been affected. I have provisionally removed those two percentile metrics in the test until the backend is adapted.

File: README.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ The register method supports the following parameters.
 |jsErrors|Boolean|Support js errors monitoring|false|true|
 |apiErrors|Boolean|Support API errors monitoring|false|true|
 |resourceErrors|Boolean|Support resource errors monitoring|false|true|
-|useFmp|Boolean|Collect FMP (first meaningful paint) data of the first screen|false|false|
+|useFmp|Boolean|Collect FMP (first meaningful paint) data of the first screen. Deprecated: This is no longer recommended. Please use the useWebVitals instead. |false|false|
```
Comment (wu-sheng) [inline README.md:55]: Does this affect the OAP analysis and metrics?

Comment (Fine0830) [reply README.md:55]: Yes, I moved the FMP metric from` /browser/perfData` to`/browser/perfData/webVitals`.

**Comment by wu-sheng** [comment]
So, e2e should be as same as previous one, right? We just need to support new URIs.

**Comment by Fine0830** [comment]
> So, e2e should be as same as previous one, right? We just need to support new URIs.

Right. Please recheck this.

**Review by wu-sheng — APPROVED** [review]

**Pull request merged**